src/main/scala/mesosphere/marathon/DebugConf.scala

```diff
@@ -106,4 +119,29 @@ class DebugModule(conf: DebugConf) extends AbstractModule {
     val behaviors = (tracingBehavior :: metricsBehavior :: Nil).flatten
     if (behaviors.nonEmpty) bindInterceptor(MarathonMatcher, Matchers.any(), behaviors: _*)
   }
+
+  private def configureLogstash(hostPort: String) {
+    val context = LoggerFactory.getILoggerFactory.asInstanceOf[LoggerContext]
+
+    val encoder = new net.logstash.logback.encoder.LogstashEncoder()
+    encoder.setContext(context)
+    encoder.addProvider(new ArgumentsJsonProvider())
+    encoder.start()
+
+    val logstashAppender = new LogstashTcpSocketAppender()
```

Comment:
I'm sorry I forget about it. I'll add another flag `--logstash_scheme` with `udp`,`tcp` options or (I think better) modify `--logstash` to accept `[udp|tcp|ssl]://<host>[:<port>]`.